packages/@uppy/locales/src/cs_CZ.js

```diff
@@ -142,7 +142,10 @@ cs_CZ.pluralize = function pluralize (n) {
   if (n === 1) {
     return 0
   }
-  return 1
+  if (n > 1 && n < 5) {
```

Comment:
`0` needs to return 1, right?

```suggestion
  if (n < 5) {
```

Reply:
Nevermind, I read this wrong.